content/docs/introduction/billing.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Billing
+enableTableOfContents: true
+isDraft: true
+---
+
+Neon offers the following plans: **Free Tier**, **Pro**, **Enterprise**, and **Platform Partnership**. The Pro plan is _usage-based_, ensuring you never over-provision and only pay for what you use. The **Enterprise** and **Platform Partnership** plans are custom volume-based plans that offer potential discounts. You can find out more about our [plans](#neon-plans) below.
+
+## Neon billing metrics
+
+Neon's charges for usage based on the following metrics:
+
+- **Compute time**: The amount time that Neon compute resources are active.
+- **Project storage**: The amount of data stored in your Neon projects.
```

Comment:
Project storage captures both data and history, this should be reflected in the definition.

Reply:
```suggestion
- **Project storage**: The size of the data and history stored for your project.
```